src/client/components/editor/title.js

```diff
@@ -1,28 +1,77 @@
 import h from 'react-hyperscript';
+import { Component } from 'react';
 
+import Popover from '../popover/popover';
+import RequestForm from '../request-form';
 import { DOI_LINK_BASE_URL } from '../../../config';
 
-export const EditorTitle = props => {
-  const { authors: { abbreviation }, title = 'Unnamed document', reference, doi } = props.citation;
-  const { document } = props;
+class EditorTitle extends Component {
+  constructor(props){
+    super(props);
 
-  if( !document.editable() ){
-    return null;
+    const { bus, document } = this.props;
+    const citation = document.citation();
+
+    this.bus = bus;
+
+    this.state = {
+      citation
+    };
+
+    this.onRequestBtnClick = () => this.setState({ citation: document.citation() });
+  }
+
+  componentDidMount(){
+    this.bus.on('requestBtnClick', this.onRequestBtnClick);
+  }
+
+  componentWillUnmount(){
+    this.bus.removeListener('requestBtnClick', this.onRequestBtnClick);
   }
-  
-  return h('div.editor-title', [
-    h('div.editor-title-content', [
-      h(doi ? 'a' : 'div', (doi ? { target: '_blank', href: `${DOI_LINK_BASE_URL}${doi}` } : {}), [
-        h('div.editor-title-name', [
-          h('span' + (doi ? '.plain-link.link-like' : ''), title)
-        ]),
-        h('div.editor-title-info', [
-          h('div', abbreviation ),
-          h('div', reference )
+
+  render(){
+    const { document } = this.props;
+    const { citation } = this.state;
+    const { authors: { abbreviation }, title = 'Unnamed document', reference, doi } = citation;
+
+    if( !document.editable() ){
+      return null;
+    }
+
+    const getTitleContent = () => {
+       if ( title == null ) {
```

Comment:
What are the conditions under which I'm allowed to edit title? I don't think `null` is even possible?

Reply:
It might be better as `if(title)` to handle the empty string case.  It should be possible to have an empty title as a part of the email campaign.  We send a completely empty document to a user so that they can try it out and fill in the paper details after the fact (#901).

Reply:
I'm missed this discussion  but I thought the point was to always make the info editable?

Reply:
The main editable case is for the when you start with an empty doc (e.g. related papers emails).  Is there another case that would benefit from editing that info after the fact?  If all the other cases have the info already set, is there a benefit of being able to change those values?

I suppose you could have an author's paper who changed the title of the paper slightly, so they want to update the factoid.  That's probably a rare edge case.

Reply:
I actually set the default value for the title as ``null`` instead of ``Untitled`` to better understand if it is actually set (https://github.com/PathwayCommons/factoid/pull/913/files#diff-bbf58575632e23c789ad93390b6495ff736ac319e4ee5e2473416a83e1bd03caR230).